.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Hey @donaldpipowitch ,

Any info on what this vscode setting does?

Reply:
Hi @boyney123,

sorry, this change accidentally slipped in. I can revert it, if you want.
But in case you use VS Code... this one is actually quite handy, I use it basically on every of my own projects. By setting `"typescript.tsdk"` you tell VS Code which TypeScript version should be used for the Language Server. It is either the one build into VS Code or the one belonging to the project. By not relying on the VS Code internal TypeScript version (which could be newer or older then the TypeScript version referenced in your project) you should always see the same errors in your editor as you see it when you type check your project via the terminal.

Reply:
Cool thanks for explaining 👍 

Yeah let's keep it in then, sounds good to me!